styles/styles.css

```diff
@@ -186,17 +199,25 @@ header {
     margin: 0 auto;
   }
 
-  main .section .default-content-wrapper {
-    max-width: var(--section-desktop-max-width);
-    padding: 0 var(--section-desktop-padding);
-    margin: 0 auto;
+  .section {
+    padding: 48px 0;
+  }
+
+  .section .default-content-wrapper {
+    padding: 0 var(--section-large-desktop-padding);
+  }
+}
+
+@media (min-width: 1400px){
+  .section {
+    padding: 60px 0;
   }
 }
 
 @media (min-width: 1600px) { /* large desktop */
-  main .section .default-content-wrapper {
+  .section .default-content-wrapper {
```

Comment:
Please keep the `main .section` to remain consistent with how other rules are written wrt section

Reply:
done